README.md

```diff
@@ -21,19 +22,20 @@ Principally, this repository contains the federated learning (FL) engine aimed a
     - [Examples](#examples)
     - [Research Code](#research-code)
     - [Tests](#tests)
-    - [Development Practices](#development-practices)
-        - [Development Requirements](#development-requirements)
-        - [Coding Guidelines, Formatters, and Checks](#coding-guidelines-formatters-and-checks)
-        - [Running Tests](#running-tests)
     - [Citation](#citation)
 
-<!-- /TOC -->
-
-The library source code is housed in the `fl4health` folder. This library is built on the foundational components of Flower, an open-source FL library in its own right. The documentation is [here](https://flower.dev/docs/framework/index.html). This library contains a number of unique components that extend the functionality of Flower in a number of directions.
+The library source code is housed in the `fl4health` folder. This library is built on the foundational components of [Flower](https://flower.dev/), an open-source FL library in its own right. The documentation is [here](https://flower.dev/docs/framework/index.html). This library contains a number of unique components that extend the functionality of Flower in a number of directions.
 
 ## Summary of Approaches
 
 The present set of FL approaches implemented in the library are:
+<table>
```

Comment:
major nit: I really like the idea of a table but this one looks a bit awkward to me. I was thinking of maybe having each method in a row. Then also a column with some sort of short description. Then if there are variants, we can mention and link in the notes.

As for splitting up into personal vs non-personal you could have a a row dedicated as the title for non personal then list the methods and then after another row dedicated as the title for personal then list the methods. 

Again, maybe this is just personal preference, so feel free to ignore

Reply:
Done